src/uws_client.c

```diff
@@ -1356,10 +1477,16 @@ static void on_underlying_io_bytes_received(void* context, const unsigned char*
                                 size_t pong_frame_length;
                                 BUFFER_HANDLE pong_frame_buffer;
 
-                                uws_client->uws_state = UWS_STATE_ERROR;
```

Comment:
Worth calling out that I removed this line. It seems like a bug that we are setting the state to error when receiving a ping frame, but I could be missing something.

Reply:
I talked to the implementer and you are correct.

---
In reply to: [188471094](https://github.com/Azure/azure-c-shared-utility/pull/156#discussion_r188471094) [](ancestors = 188471094)